roles/openshift_loadbalancer/templates/haproxy.cfg.j2

```diff
@@ -80,3 +79,33 @@ backend {{ backend.name }}
     server      {{ server.name }} {{ server.address }} {{ server.opts }}
 {% endfor %}
 {% endfor %}
+
+{% if openshift_use_contrail | default(false) | bool %}
```

Comment:
We should not use the apiserver load balancer to serve contrail UI. This should likely be its own load balancer.

Reply:
Hi @rajatchopra will remove this section. If we want to provide HA services for Contrail, then do you suggest bringing in our own loadbalancer node?